yaksh/code_server_settings.py

```diff
@@ -0,0 +1,51 @@
+"""
+settings for code_server
+"""
+
+from decouple import config
+
+# The number of code server processes to run..
+N_CODE_SERVERS = config('N_CODE_SERVERS', default=5, cast=int)
+
+# The server pool port.  This is the server which returns available server
+# ports so as to minimize load.  This is some random number where no other
+# service is running.  It should be > 1024 and less < 65535 though.
+SERVER_POOL_PORT = config('SERVER_POOL_PORT', default=55555, cast=int)
+
+SERVER_HOST_NAME = config('SERVER_HOST_NAME', default='http://localhost')
+
+# Timeout for the code to run in seconds.  This is an integer!
+SERVER_TIMEOUT = config('SERVER_TIMEOUT', default=4, cast=int)
+
+code_evaluators = {
```

Comment:
not sure why this was moved from yaksh/settings.py to code_server_settings.py? Is the modification only for better understanding of the intent of the file? Or does this change some core functionality?

Reply:
Just for better understanding, felt that its more closer to code_server.
It does not change the core functionality.